examples/torch/common/example_logger.py

```diff
@@ -2,6 +2,7 @@
 import sys
 
 logger = logging.getLogger("example")
+logger.propagate = False
```

Comment:
What is it change?

Reply:
I find out regularly that some Python packages such as `rich` try to set up custom handlers to the global logger that suppress its output. If `.propagate` is not False in such a case, then it looks like the "example" logger will pass the logs to the global logger, whose handlers will then just ignore the logs, and we end up in a situation where no example logs are visible, which is off-putting.